### DISCUSSION TITLE
Publishing section-properties at https://joss.theoj.org/

### DISCUSSION THREAD

**Discussion by simo-11** [robbievanleeuwen/section-properties#223] [General]
Have you considered publishing section-properties at Journal of Open Source Software?
I would make referring easy and also promote section-properties.

**Comment by robbievanleeuwen** [comment]
@simo-11 thanks for the suggestion, I'll look into this!

**Comment by robbievanleeuwen** [comment]
Took me a while - but we got there :)

https://joss.theoj.org/papers/10.21105/joss.06105
